supergrub-mkcommon

```diff
@@ -66,21 +66,52 @@ function generate_and_copy_mo_files () {
 }
 
 function check_unifont () {
+  local unifont_basename="unifont.pf2"
+  local unifont_destdir="$overlay/boot/grub/fonts"
+  local unifont_destfile="$unifont_destdir/$unifont_basename"
+  mkdir -p -- "$unifont_destdir" || return $?
+
+  local font_dirs="$(echo $(echo '
+    /boot/grub/fonts
+    /usr/share/grub/fonts
+    '))"
+  local dir= ext=
```

Comment:
Why two nested echos?

Why not just:
```
local font_dirs="/boot/grub/fonts /usr/share/grub/fonts"
```
?

Reply:
In case we add more search locations in the future, this style will cause less diff flickering, and allows easier sorting of items.

Reply:
I think we do not need those additional echos.

If you want less diff flickering you can use:
```
  local font_dirs="\
    /boot/grub/fonts \
    /usr/share/grub/fonts \
"
```
instead.